ietfYangDraftPull/draftPull.py

```diff
@@ -116,11 +117,22 @@ def main(script_conf: ScriptConfig = DEFAULT_SCRIPT_CONFIG.copy()) -> list[dict[
                 os.chmod(rfc_exceptions, 0o664)
                 remove_from_new = []
             new_files = [file_name for file_name in new_files if file_name not in remove_from_new]
-
+            automatic_pull_request_creation_result = create_new_rfcs_pull_request(
```

Comment:
As we discussed earlier we will just create and push the commit to the repo. API endpoint is responsible for creating Pull Request, so we should not use `pull_request` term here at all.